extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/dispatcher/atlas/AtlasClientConf.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.plugin.lineage.dispatcher.atlas
+
+import org.apache.atlas.ApplicationProperties
+import org.apache.commons.configuration.Configuration
+import org.apache.spark.kyuubi.lineage.SparkContextHelper
+
+import org.apache.kyuubi.plugin.lineage.dispatcher.atlas.AtlasClientConf.ConfigEntry
+
+class AtlasClientConf(configuration: Configuration) {
+
+  def get(entry: ConfigEntry): String = {
+    configuration.getProperty(entry.key) match {
+      case s: String => s
+      case l: List[_] => l.mkString(",")
+      case o if o != null => o.toString
+      case _ => entry.defaultValue
+    }
+  }
+
+}
+
+object AtlasClientConf {
+
+  private lazy val clientConf: AtlasClientConf = {
+    val conf = ApplicationProperties.get()
+    SparkContextHelper.globalSparkContext.getConf.getAllWithPrefix("spark.atlas.")
+      .foreach {
+        case (k, v) => conf.setProperty(s"atlas.$k", v)
+      }
```

Comment:
```suggestion
      .foreach { case (k, v) => conf.setProperty(s"atlas.$k", v) }
```
Minor. In a compact code style.